README.md

```diff
@@ -73,5 +73,14 @@ $ docker-compose -f docker-compose.prod.yml up
 ```
 It will serve backend at port 8080 and frontend at port 80.
 
+
+## Mobile
```

Comment:
Mobile section has some prerequisites that aren't listed on the top of the README.
As far as I know these tools are necessary:
- npm
- yarn
- expo

Also the user has to be in "mobile" directory and run:
```
npm install
```
before running 'yarn start'